modules/lemmaBidAdapter.js

```diff
@@ -0,0 +1,406 @@
+import * as utils from '../src/utils';
+import { registerBidder } from '../src/adapters/bidderFactory';
+import { BANNER, VIDEO } from '../src/mediaTypes';
+
+var BIDDER_CODE = 'lemma';
+var LOG_WARN_PREFIX = 'LEMMA: ';
+var ENDPOINT = '//ads.lemmatechnologies.com/lemma/servad';
+var DEFAULT_CURRENCY = 'USD';
+var AUCTION_TYPE = 2;
+var DEFAULT_TMAX = 300;
+var DEFAULT_NET_REVENUE = false;
+
+export var spec = {
+
+  code: BIDDER_CODE,
+  supportedMediaTypes: [BANNER, VIDEO],
+
+  isBidRequestValid: bid => {
+    if (bid && bid.params) {
+      if (utils.isStr(bid.params.pubId) || !bid.params.pubId) {
```

Comment:
What should the type of `pubId` be? Validation should check that it's the correct type instead of that it's not a string.

If `pubId` needs to be a number, the check should be `!utils.isNumber(bid.params.pubId)`

Also, this way the 2nd `or` condition shouldn't be needed.

Reply:
pubId type is integer. I have updated condition and also remove 2nd condition.